fatukunda/fashion-cloud-cache-api
# ft(cache): Create cache

#### What does this PR do?
 - Creates a Cache
#### Description of Task to be completed?
 - Write tests for creating a cache.
 - Create the Cache model and define its attributes.
 - Create the cache controller, router, and service.
#### How should this be manually tested?
 - Navigate to `POST localhost:3000/cache` and provide the cache `key` in the body.
 - Hit send and if all goes well, You should see the cache data returned.
#### What are the relevant pivotal tracker stories?
[#170817488](https://www.pivotaltracker.com/story/show/170817488)
#### Any background context you want to add?
 - None
#### Screenshots 
<img width="1137" alt="Screen Shot 2020-01-21 at 2 59 19 PM" src="https://user-images.githubusercontent.com/4363632/72803243-a8837000-3c5e-11ea-8dd1-0d5d5552376d.png">
